### ISSUE TITLE
investigate pushing to alternate public repos

### ISSUE THREAD

**Issue by xquery** [curl/curl-docker#66]
github container registry, quay, etc

**Comment by DragonStuff** [comment]
Would be great to upload the official curl images to AWS ECR Public: https://gallery.ecr.aws/

**Comment by onedr0p** [comment]
Given the recent dockerhub announcement of removing free team orgs, I'm wondering if the current org will be affected? And if so perhaps this could finally have a high priority?

https://github.com/docker/hub-feedback/issues/2314

**Comment by xquery** [comment]
yes we are currently gathering up information and have been (before this announcement) investigating what other registries to publish to ... with quay.io and github being candidates.
